src/components/sbb-tab-group/sbb-tab-group.tsx

```diff
@@ -0,0 +1,325 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Method,
+  Prop,
+} from '@stencil/core';
+import { InterfaceSbbTabGroupTab } from './sbb-tab-group.custom';
+import { AgnosticMutationObserver as MutationObserver } from '../../global/helpers/mutation-observer';
+import { AgnosticResizeObserver as ResizeObserver } from '../../global/helpers/resize-observer';
+import { hostContext } from '../../global/helpers/host-context';
+import throttle from '../../global/helpers/throttle';
+import getDocumentWritingMode from '../../global/helpers/get-document-writing-mode';
+
+/**
+ * @slot tab-bar - Provide an `sbb-tab-title` in order to display a label in the tab bar.
```

Comment:
The consumer doesn't have to use the tab-bar slot directly. should it really appear here? or should we document somehow?